en/components/banner.md

```diff
@@ -26,7 +26,7 @@ The Banner has styling flexibility with the many options for the background, bor
 
 ## Usage
 
-When using a Banner...
+When using a Banner make sure your elements and texts are visible and readable on the background.
```

Comment:
look here and pick the 2-3  most applicable to our banner do-dont scenarios
https://material.io/components/banners#anatomy
about the images you may want to insert, just add the img tags with correct names and then archive and send them over to me to add them via source tree. there is such an image tag in the beginning of the topic